modules/geo-layers/src/terrain-layer/terrain-layer.js

```diff
@@ -146,7 +146,7 @@ export default class TerrainLayer extends CompositeLayer {
   renderSubLayers(props) {
     const SubLayerClass = this.getSubLayerClass('mesh', SimpleMeshLayer);
     const {data, color} = props;
-    const [mesh, texture] = data;
+    const [mesh, texture] = data || [];
```

Comment:
Just return null if data is not available?

Reply:
Oh duh that does make more sense